app/src/main/AndroidManifest.xml

```diff
@@ -23,81 +24,75 @@
-                  android:theme="@android:style/Theme.Dialog"
-                  android:launchMode="singleInstance"
-                  android:excludeFromRecents="true"
-                  android:finishOnTaskLaunch="true" />
-
+        android:supportsRtl="true"
+        android:theme="@style/LightAppTheme">
+        <activity
+            android:name="org.acra.CrashReportDialog"
+            android:excludeFromRecents="true"
+            android:finishOnTaskLaunch="true"
+            android:launchMode="singleInstance"
+            android:theme="@android:style/Theme.Dialog" />
         <activity android:name=".auth.LoginActivity">
             <intent-filter>
                 <category android:name="android.intent.category.LAUNCHER" />
+
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
-
         <activity android:name=".WelcomeActivity" />
-
         <activity
             android:name=".upload.ShareActivity"
             android:icon="@drawable/ic_launcher"
             android:label="@string/app_name">
             <intent-filter>
                 <action android:name="android.intent.action.SEND" />
+
                 <category android:name="android.intent.category.DEFAULT" />
+
                 <data android:mimeType="image/*" />
                 <data android:mimeType="audio/ogg" />
             </intent-filter>
         </activity>
-
         <activity
             android:name=".upload.MultipleShareActivity"
             android:icon="@drawable/ic_launcher"
             android:label="@string/app_name">
             <intent-filter>
                 <action android:name="android.intent.action.SEND_MULTIPLE" />
+
                 <category android:name="android.intent.category.DEFAULT" />
+
                 <data android:mimeType="image/*" />
                 <data android:mimeType="audio/ogg" />
             </intent-filter>
         </activity>
-
         <activity
             android:name=".contributions.ContributionsActivity"
             android:icon="@drawable/ic_launcher"
             android:label="@string/app_name" />
-
         <activity
             android:name=".settings.SettingsActivity"
             android:label="@string/title_activity_settings" />
-
         <activity
             android:name=".AboutActivity"
             android:label="@string/title_activity_about"
             android:parentActivityName=".contributions.ContributionsActivity" />
-
         <activity
             android:name=".auth.SignupActivity"
             android:label="@string/title_activity_signup" />
-
         <activity
             android:name=".nearby.NearbyActivity"
             android:label="@string/title_activity_nearby"
             android:parentActivityName=".contributions.ContributionsActivity" />
-
         <activity
             android:name=".notification.NotificationActivity"
             android:label="@string/navigation_item_notification" />
-
```

Comment:
Please undo all such formatting changes that don't apply to this diff.

Reply:
@maskaravivek Sure, I will undo all such changes, sorry for a bad quality pull request, I am new to open source :disappointed:

Reply:
Don't feel bad. Most of us need to make changes to our PRs before they are accepted. ;)

Reply:
Yup @madhurgupta10 you have really done a great job with this PR. I personally wanted this feature in the app as soon as possible and with your PR we are very close to it. :)

Reply:
@maskaravivek @misaochan Thanks, I will update the PR asap.